doc/classes/Node.xml

```diff
@@ -983,7 +983,7 @@
 			Never process. Completely disables processing, ignoring the [SceneTree]'s paused property. This is the inverse of [constant PROCESS_MODE_ALWAYS].
 		</constant>
 		<constant name="DUPLICATE_SIGNALS" value="1" enum="DuplicateFlags">
-			Duplicate the node's signals.
+			Duplicate the node's outgoing signals that have been set with the CONNECT_PERSIST flag. See [enum Object.ConnectFlags] for possible connection flags.
```

Comment:
```suggestion
			Duplicate the node's outgoing signals that have been set with the [constant Object.CONNECT_PERSIST] flag. See [enum Object.ConnectFlags] for possible connection flags.
```

Reply:
And then the "See [enum ...]" may be redundant I guess?

Reply:
Thanks, I actually wanted to do this but wasn't completely sure how to and if it would work